webroot/src/models/AdverseAction/AdverseAction.model.ts

```
@@ -0,0 +1,118 @@
+            providerId: json.providerId,
+            state: new State({ abbrev: json.jurisdiction }),
+            type: json.type,
+            npdbType: json.clinicalPrivilegeActionCategory,
+            creationDate: json.creationDate,
+            startDate: json.effectiveStartDate,
+            endDate: json.effectiveLiftDate,
+        };
+
+        return new AdverseAction(adverseActionData);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)